tasks/blacklight.rake

```diff
@@ -52,8 +52,8 @@ namespace :blacklight do
       Rake::Task['engine_cart:generate'].invoke
     end
 
-    SolrWrapper.wrap(port: '8983') do |solr|
-      solr.with_collection(name: 'blacklight-core', dir: File.join(File.expand_path("..", File.dirname(__FILE__)), "solr", "conf")) do
+    SolrWrapper.wrap(version: '7.1.0', port: '8983') do |solr|
```

Comment:
Is this parameter change required?

Reply:
Not sure, probably not.  I included to be consistent with the CI setup.  See line 16 above.